lib/utils/fast-boot.js

```diff
@@ -0,0 +1,22 @@
+const { getParent } = require('../utils/common');
+
+function isEnvironmentBrowser(node) {
+  let callee = node.test.callee;
+
+  if (callee) {
+    let obj = callee.object;
+    let prop = callee.property;
+
+    return obj.name === 'environment' && prop.name === 'isBrowser';
```

Comment:
Since this addon is open source and not internal, we should check for isFastBoot too. `environment.isBrowser()` is internal to us. 

This addon should be provide an API to add additional lint rules from another app/addon. That way you can add lint rule for `environment.isBrowser()` for internal apps while continue to check for `isFastBoot` for open source.